src/main/java/dk/cngroup/lentils/exception/DirectoryCanNotBeCreatedException.java

```diff
@@ -0,0 +1,7 @@
+package dk.cngroup.lentils.exception;
+
+public class DirectoryCanNotBeCreatedException extends SecurityException {
```

Comment:
@JanaCNGroup Má to nějaký důvod proč to dědí ze SecurityException a ne třeba jen z RuntimeException?
@mjanousek

Reply:
našla jsem si, že mkdirs() vyhazuje SecurityException, tak jsem myslela, že by to mělo dědit z ní. Ale teď se dívám, že všechny Exceptions, které na Luštěninách máme, dědí z 
RuntimeException. A je správně, když v metodě, která tuto Exception vyhazuje, ji mám odchycenou taky jako SecurityException? (ImageService - metoda getDirectory(), ř. 113)